contribs/gmf/src/directives/objectediting.js

```diff
@@ -936,7 +936,8 @@ gmf.ObjecteditingController.prototype.handleSketchFeaturesAdd_ = function(evt) {
     }
 
   } else if (this.process === gmf.ObjecteditingtoolsController.ProcessType.ADD) {
-    this.feature.setGeometry(sketchGeom.clone());
+    this.feature.setGeometry(
+      gmf.ObjecteditingController.toMultiGeometry_(sketchGeom.clone()));
```

Comment:
It's not that I speak about ...
It the result of e.g.: https://geomapfish-demo.camptocamp.net/2.1/wsgi/layers/111/md.xsd
Is a geometry type: `gml:LineStringPropertyType` instance of `gml:MultiLineStringPropertyType` we shouldn't use a multi geometry :-)